src/css/styles.css

```diff
@@ -1,3 +1,105 @@
+body {
+    font-family: 'Lato';
+}
+
+.title {
+    text-align: center;
+    font-weight: bold;
+    font-size: 40px;
+    margin-bottom: -10px;
+}
+
+.buttonContainer {
+    text-align: right;
+    margin-right: 5%;
+    margin-bottom: 10px;
+}
+
+.timelineButton {
+    background-color: white;
+    border-radius: 12px;
+    border: 1px solid;
+    color: black;
+    font-size: 15px;
+    -webkit-transition-duration: 0.4s;
+    transition-duration: 0.4s;
+    justify-content: center;
+    padding: 5px 10px;
+}
+
+.timelineButton:hover {
+    background-color: #008CBA;
+    color: white;
+}
+
+.data {
+    text-align: center;
+    border: 1px solid;
+    border-color: black;
+    width: 90%;
+    margin: auto;
+    font-size: 25px;
+}
+
+.data:empty {
+    display: none;
+}
+
+.errorMessage {
+    color: red;
+}
+
 .item:nth-child(even) {
-    background-color: grey;
+    background-color: #e9e9e9;
+}
+
+.item {
+    background-color: #e8f5fd;
+    align-items: center;
+    border: 1px solid;
+    border-color: #C0C0C0;
+    border-top: 0;
+    border-left: 0;
+    border-right: 0;
+    display: grid;
+    grid-template-columns: 12% auto;
```

Comment:
do you need both this and the inline-block style below?

Reply:
Removed inline